homeassistant/components/conversation/trigger.py

```diff
@@ -122,4 +137,6 @@ async def call_action(
         # two trigger copies for who will provide a response.
         return None
 
-    return hass.data[DATA_DEFAULT_ENTITY].register_trigger(sentences, call_action)
+    return get_agent_manager(hass).default.register_trigger(
```

Comment:
it feels weird that we would be directly calling things on `get_agent_manager(hass).default`.